sshd/issue_invite.sh

```diff
@@ -28,36 +28,37 @@ done
 shift $((OPTIND-1))
 
 TMP=`mktemp -d`
+npm install jsurl &>/dev/null
```

Comment:
Tried this (put `npm install -g jsurl` below the apt-gets in the Dockerfile), the result was "Cannot find module 'jsurl'". Would placing it elsewhere in the Dockerfile maybe work?